README.md

```diff
@@ -54,6 +54,20 @@ Stopping local Kubernetes cluster...
 Stopping "minikubeVM"...
 ```
 
+### Reusing the Docker daemon
+
+When using a single VM of kubernetes its really handy to reuse the Docker daemon inside the VM; as this means you don't have to build on your host machine and push the image into a docker registry - you can just build inside the same docker daemon which speeds up local experiements.
+
+To be able to work with the docker daemon on your mac/windows host type the following into your shell:
```

Comment:
We don't currently support windows, remove for now? We support linux so you could swap it to that.